src/components/ApartmentApplicantList/components/ApplicantListItem/index.js

```diff
@@ -8,9 +8,15 @@ import {
   ListItemSecondaryAction,
   ListItemText,
   Avatar,
-  IconButton,
+  Button,
+  Menu,
+  MenuItem,
+  ListItemIcon,
 } from '@material-ui/core';
 import PersonIcon from '@material-ui/icons/Person';
+import ArrowDropDownIcon from '@material-ui/icons/ArrowDropDown';
+import StarBorderIcon from '@material-ui/icons/StarBorder';
+// import ContactMailIcon from '@material-ui/icons/ContactMail';
```

Comment:
Are you planning to use this import later on? Or can we just delete it?

Reply:
I do not remember what I used that icon for at the time, but I know it is no longer needed in that file. So we can delete that import